src/esw/brushless.py

```diff
@@ -0,0 +1,231 @@
+#!/usr/bin/env python3
+import threading
+import time as t
+import asyncio
+import rospy
+import math
+from sensor_msgs.msg import JointState
+from geometry_msgs.msg import Twist
+from std_msgs.msg import Header
+import numpy as np
+import moteus
+
+
+# import moteus_pi3hat
+
```

Comment:
What was the justification for not including the Pi3hat functionality in this PR? It is implemented in ato/brushless-drive.

Reply:
Ok in my most recent commit, I included pi3hat functionality. I just have it commented it out so you can still run the program on your laptop (it's commented out since ur laptop can't install moteus pi3hat so it will error out).

The code I added looks like this::

```python
using_pi3_hat = rospy.get_param("brushless/using_pi3_hat")
        if using_pi3_hat:
            # import moteus_pi3hat
            # transport = moteus_pi3hat.Pi3HatRouter(
            #     servo_bus_map={
            #         1: [11],
            #         2: [12],
            #         3: [13],
            #         4: [14],
            #     },
            # )
            pass
        else:
            transport = moteus.Fdcanusb()

        for name, info in drive_info_by_name.items():
            self.drive_bridge_by_name[name] = MoteusBridge(info["id"], transport)
```

Reply:
Ok yea, The pi3hat functionality shouldn't be a lot to add. You will have to add the bash script I made for running this using sudo when on the Pi. (Unless there is a better way, but that is what has been working. It's under the scripts folder in ato/brushless-drive) Otherwise that is alright. I agree it will be easier for testing individual buses. Later on though we should move to using the Pi. Especially since we'll want to run drive and arm code concurrently.